web/BricksAndHearts.csproj

```diff
@@ -9,8 +9,9 @@
 
     <ItemGroup>
         <PackageReference Include="Azure.Storage.Blobs" Version="12.13.0" />
-        <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.21.0" />
         <PackageReference Include="MailKit" Version="3.3.0" />
+        <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.21.0" />
+        <PackageReference Include="jQuery" Version="3.6.0" />

```

Comment:
Are we using this? I don't think we need jQuery as a C# package, we have a copy in `wwwroot/js/lib`